heartbeat/monitors/active/icmp/loop.go

```diff
@@ -92,13 +94,25 @@ func newICMPLoop() (*icmpLoop, error) {
 	// IPv4/IPv6 checking
 	conn4 := createListener("IPv4", "ip4:icmp")
 	conn6 := createListener("IPv6", "ip6:ipv6-icmp")
-
+	unprivilegedPossible := false
 	l := &icmpLoop{
 		conn4:    conn4,
 		conn6:    conn6,
 		recv:     make(chan packet, 16),
 		requests: map[requestID]*requestContext{},
 	}
+	if conn4 == nil && conn6 == nil {
+
```

Comment:
nit: newline before this if, but not after.

Reply:
thanks.. changing..